corehq/apps/reports/static/reports/ko/export.manager.js

```diff
@@ -176,6 +176,7 @@ var ExportManager = function (o) {
             url: downloadUrl,
             type: 'POST',
             data: data,
+            traditional: true,
```

Comment:
cool, never knew about that and had found it annoying in the past (`QueryDict` kind of handles this for you which is maybe why we weren't running into larger issues)